ctapipe_io_lst/event_time.py

```diff
@@ -141,6 +142,7 @@ class EventTimeCalculator(TelescopeComponent):
         default_value=None,
         help='UCTS timestamp of a valid ucts/tib counter combination'
     ).tag(config=True)
+
```

Comment:
I would add an help to the trailet 
`timestamp = TelescopeParameter(
        trait=Enum(['ucts', 'dragon', 'tib']), default_value='dragon'
    ).tag(config=True)`

the name is not enough self-explaining

Reply:
In general, it would be better to add to the doc of the class a short description of the logics that it contains

Reply:
both done